src/product-entry.js

```diff
@@ -0,0 +1,20 @@
+import store from './store.js';
+
+const form = document.getElementById('new-hat');
+
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const formData = new FormData(form);
+    const newHat = {
+        code: formData.get('code'),
+        name: formData.get('name'),
+        image: formData.get('image'),
```

Comment:
You need to do more here than just get the image form input - take a look at the getBase64 stuff in marty's sample code to see how he handled this. Right now, your code throws a 404 Not Found when trying to load the image when you view the newly created product in the store.

Reply:
I think I got it worked out now.